CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable, unreleased changes to this project will be documented in this file.
 
 # 3.1.0 [Unreleased]
 
+- Add Click&Collect feature - #SALEOR-2992 by @kuchichan
```

Comment:
In the public changelog, we want to refer to PRs that introduced the described changes.
```suggestion
- Add Click&Collect feature - #7673 by @kuchichan
```

Reply:
🥇  Done.